components/Tracer/index.tsx

```diff
@@ -28,11 +28,14 @@ export interface TraceEntry {
   fp: number
 }
 
+export type Variables = { [key: string]: Array<string> }
```

Comment:
Can we call it `SierraVariables` to make it more explicit? Also will be easier to differentiate once we add `Variables` for Cairo or CASM.